QRCoder/QRCodeGenerator.cs

```diff
@@ -802,20 +880,20 @@ public static int Score(ref QRCodeData qrCode)
                                 blackModules++;
 
                     var percent = (blackModules / (qrCode.ModuleMatrix.Count * qrCode.ModuleMatrix.Count)) * 100;
-                    var prevMultipleOf5 = Math.Abs((int) Math.Floor(percent/5)*5 - 50)/5;
-                    var nextMultipleOf5 = Math.Abs((int)Math.Floor(percent / 5) * 5 -45)/5;
-                    score4 = Math.Min(prevMultipleOf5, nextMultipleOf5)*10;
+                    var prevMultipleOf5 = Math.Abs((int)Math.Floor(percent / 5) * 5 - 50) / 5;
+                    var nextMultipleOf5 = Math.Abs((int)Math.Floor(percent / 5) * 5 - 45) / 5;
+                    score4 = Math.Min(prevMultipleOf5, nextMultipleOf5) * 10;
 
                     return score1 + score2 + score3 + score4;
```

Comment:
```suggestion
                    return (score1 + score2) + (score3 + score4);
```
See [sharplab](https://sharplab.io/#v2:EYLgxg9gTgpgtADwGwBYA0AXEBDAzgWwB8ABAJgEYBYAKGIGYACMhgYQYG8aHunGATCAFdgAGxgMAsuQAUA4WIa5IscmgZzR4pdBik1Ghdth19QzYuUwUASi49O1Hk6YB2CzvIMA1O9ilvvjCMPkZWANx23AC+kQyx9OpmChKksklalqqJ8hk6etnmoSYFhpY2sQ7OPMRu0qGeIZak1gF1lsGBNhGOPDHUUUA===).

With the paranthesis it breaks the register dependency on `xmm0`, and the cpu can utilize more parallelism.